server/backend-api/app/api/routes/attendance.py

```
@@ -124,33 +124,31 @@ async def mark_attendance(payload: Dict):
 
     # Build results
     results = []
-    
-    print(f"Faces detected: {len(detected_faces)}")
+    logger.info("Faces detected: %d", len(detected_faces))
 
     for i, (face, match) in enumerate(zip(detected_faces, matches)):
```

Ruff (0.14.14)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)